shared/testutil/block.go

```diff
@@ -569,30 +541,26 @@ func NewPandoraBlock(
 		Nonce:       gethTypes.BlockNonce{0x01, 0x02, 0x03},
 	}, nil, nil, nil, nil)
 
-	return block.Header(), sealHash(block.Header()), &extraData
+	return block.Header(), &extraData
 }
 
-// sealHash returns the hash of a block prior to it being sealed.
-func sealHash(header *eth1Types.Header) (hash common.Hash) {
-	hasher := sha3.NewLegacyKeccak256()
-
-	if err := rlp.Encode(hasher, []interface{}{
-		header.ParentHash,
-		header.UncleHash,
-		header.Coinbase,
-		header.Root,
-		header.TxHash,
-		header.ReceiptHash,
-		header.Bloom,
-		header.Difficulty,
-		header.Number,
-		header.GasLimit,
-		header.GasUsed,
-		header.Time,
-		header.Extra,
-	}); err != nil {
-		return eth1Types.EmptyRootHash
-	}
-	hasher.Sum(hash[:0])
-	return hash
+// NewBeaconBlockWithPandoraSharding
+func NewBeaconBlockWithPandoraSharding(panHeader *gethTypes.Header, slot types.Slot) *ethpb.SignedBeaconBlock {
```

Comment:
please rename it to `NewBeaconBlockWithPandoraShard` 
Sharding is a verb, Shard is a noun. You should use noun because there is no action in block function